### ISSUE TITLE
Game blocked when Internet connection is lost

### ISSUE THREAD

**Issue by 221V** [altenwald/leprechaun#7]
the game can not be continued when internet connection was lost for 1 or few seconds((

**Comment by manuel-rubio** [comment]
Yes, I could reproduce it. It happens when the Internet or connection is gone, you perform a move and then the connection is back. The move is missing, and the server is waiting for a new move but the client thought it was sent. I need to change this in the client part to ensure the move is resent.
